.babelrc.js

```diff
@@ -9,6 +9,14 @@ module.exports = {
     ['@babel/proposal-object-rest-spread', { loose }],
     '@babel/transform-react-jsx',
     cjs && ['@babel/transform-modules-commonjs', { loose }],
-    ['@babel/transform-runtime', { useESModules: !cjs }],
-  ].filter(Boolean),
+    [
+      '@babel/transform-runtime',
+      {
+        useESModules: !cjs,
+        version: require('../package.json').dependencies[
```

Comment:
This is at the root, so it shouldn't be looking up a directory.

Reply:
right 😅this is how last-second changes end up 😉 just fixed it